main.js

```diff
@@ -1,57 +1,82 @@
-const form = document.querySelector('#form');
-const mainHolder = document.querySelector('.container');
-const bookInput = document.querySelector('#book');
+/* eslint-disable  max-classes-per-file */
+const bookList = document.querySelector('#book-list');
+const bookInput = document.querySelector('#title');
 const authorInput = document.querySelector('#author');
 const buttonAdd = document.querySelector('#add');
 
-let books = [];
-
-function show() {
-  const books = JSON.parse(localStorage.getItem('books'));
-  for (let index = 0; index < books.length; index += 1) {
-    bookInput.value = books[index].book;
-    authorInput.value = books[index].author;
+class Books {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
   }
 }
-form.addEventListener('load', () => {
-  show();
-});
 
-if (localStorage.getItem('books')) {
-  books = JSON.parse(localStorage.getItem('books'));
-}
+class SavelocalItems {
+  static getBook() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
 
-function addBook(book, author) {
-  const newBook = { book, author };
-  books.push(newBook);
-  localStorage.setItem('books', JSON.stringify(books));
+  static saveItem(book) {
+    const books = SavelocalItems.getBook();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeItem(author) {
+    const books = SavelocalItems.getBook();
+    books.forEach((book, index) => {
+      if (book.author === author) {
+        books.splice(index, 1);
+      }
+    });
+    localStorage.setItem('books', JSON.stringify(books));
+  }
 }
-function displayBooks() {
-  mainHolder.innerHTML = '';
-  for (let i = 0; i < books.length; i += 1) {
-    const bookHolder = document.createElement('section');
+
+// const booksInserted = [];
+
```

Comment:
- [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.